conflux/README.md

```diff
@@ -0,0 +1,37 @@
+# Chainlink External Adapter for Conflux Network
+
+This adapter is built to fulfill Chainlink oracle requests.
+
+## Configuration
+
+The adapter uses the following environment variables:
+
+- `URL`: A URL to a JSON-RPC (HTTP RPC) node on Conflux Network
+- `NETWORK_ID`: networkId of a JSON-RPC (HTTP RPC) node on Conflux Network
+- `PRIVATE_KEY`: The private key to sign transactions with. Must have fulfillment permissions on the Oracle contract.
+- `EA_PORT`: Configure this parameter to change the EA port
```

Comment:
`EA_PORT` is a common env var for every adapter, there's no need of specifying it